hypervisor/Makefile

```diff
@@ -42,6 +42,8 @@ include scripts/kconfig/kconfig.mk
 #initialize scenarios name
 ifeq ($(CONFIG_SDC),y)
 SCENARIO_NAME := sdc
+else ifeq ($(CONFIG_LOGICAL_PARTITIONED_2VM),y)
```

Comment:
just use logical_partition is fine. No need to say 2VM

Reply:
ok